TodoApp/tests/TodoApp.Contracts.Base/EqualityComparer/TodosEqualityComparer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using TodoApp.Contracts.Models;
+
+namespace TodoApp.Contracts.Base.EqualityComparer
+{
+    internal class TodosEqualityComparer : IEqualityComparer<Todo>
+    {
+        private static readonly Lazy<TodosEqualityComparer> _instance = new Lazy<TodosEqualityComparer>(() => new TodosEqualityComparer());
+
+        public static TodosEqualityComparer Instance => _instance.Value;
+        
+        private TodosEqualityComparer() { }
+
+        public bool Equals(Todo x, Todo y)
+        {
+            return x.Id == y.Id
+                && x.Value == y.Value
+                && x.CreatedAt == y.CreatedAt;
+        }
+
+        public int GetHashCode(Todo obj)
+        {
+            return obj.Id.GetHashCode() ^ obj.CreatedAt.GetHashCode();
```

Comment:
This could be lambda :wink:

Reply:
Read more on [hashcode](https://msdn.microsoft.com/en-us/library/system.object.gethashcode.aspx). It is OK for me to return an arbitrary value in test, however, if there is to be an implementation, better be right (especially since R# generates whole equality comparer for you)